include/random_walks/gaussian_hamiltonian_monte_carlo_exact_walk.hpp

```diff
@@ -57,8 +57,8 @@ struct Walk
     template <typename GenericPolytope>
     Walk(GenericPolytope &P, Point const& p, NT const& a_i, RandomNumberGenerator &rng)
     {
-        _Len = compute_diameter<GenericPolytope>
-                ::template compute<NT>(P);
+        //_Len = compute_diameter<GenericPolytope>
```

Comment:
why are you commenting this out? This is where `_Len` is computed that is a essential parameter of the walk.